Examples/Tests/ohm_solver_EM_modes/PICMI_inputs.py

```diff
@@ -306,7 +304,8 @@ def _record_average_fields(self):
         similar format as the reduced diagnostic so that the same analysis
         script can be used regardless of the simulation dimension.
         """
-        step = sim_ext.getistep() - 1
+        warpx = simulation.extension.warpx
+        step = warpx.getistep(lev=0) - 1
```

Comment:
I think with the new structure it is fine to change these to
```step = simulation.extension.warpx.getistep(lev=0) - 1```
and avoid the extra line. (The same in the other Ohm's law solver tests).

Reply:
that is ok, I just wrote this for readability in two lines.